tests/integration-tests/tests/cfn-init/test_cfn_init/test_install_args_quotes/post_install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "post-install script has $# arguments"
+for arg in "$@"
+do
+    echo "arg: ${arg}"
```

Comment:
Should we assert on the actual args we see to make sure the args are `R curl wget` and not `"R curl wget"`

Reply:
good point, done